src/main/python/ttconv/scc/preambles_access_codes.py

```diff
@@ -24,146 +24,162 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 """SCC Preamble Access Codes"""
-import typing
+
+from __future__ import annotations
+
+from typing import Optional
+
+from ttconv.style_properties import NamedColors, TextDecorationType, \
+  FontStyleType
+
+
+class _SccPacDescriptionBits:
+  """Helper class for SCC PAC description bits handling"""
+  def __init__(self, bits: int):
+    self._bits = bits
+
+  def get_underline(self) -> bool:
+    """Returns whether the PAC description bits sets the underline decoration"""
+    return self._bits % 2 == 1
+
+  def get_italic(self) -> bool:
+    """Returns whether the PAC description bits sets the italic style"""
+    return self._bits in [0x0E, 0x0F]
+
+  def get_color(self) -> Optional[NamedColors]:
+    """Returns the color from the PAC description bits"""
+
+    if self._bits not in list(range(0x00, 0x20)):
+      return None
+
+    if self._bits in [0x00, 0x01]:
+      return NamedColors.white
+
+    if self._bits in [0x02, 0x03]:
+      return NamedColors.green
+
+    if self._bits in [0x04, 0x05]:
+      return NamedColors.blue
+
+    if self._bits in [0x06, 0x07]:
+      return NamedColors.cyan
+
+    if self._bits in [0x08, 0x09]:
+      return NamedColors.red
+
+    if self._bits in [0x0A, 0x0B]:
+      return NamedColors.yellow
+
+    if self._bits in [0x0C, 0x0D]:
+      return NamedColors.yellow
```

Comment:
`magenta` right?

Reply:
Fixed in 681cd3a5e52267ab5ca24850bde5bc52d975cbc6